dev/Projections/CS/Microsoft.Windows.AppLifecycle/Microsoft.Windows.AppLifecycle.Projection.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.18362.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
+    <Platforms>x64;x86</Platforms>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.2.2" />
+  </ItemGroup>
+
+  <!-- For consistency across Reunion, explicitly reference .NET 5.0.5 SDK (5.0.202 train for VS 16.9.3) -->
```

Comment:
MRTCore had this which is puzzling AH as I don't see any mention of .NET 5.0.5, 5.0.202 or VS 16.9.3 in the `<FrameworkReference>`s below.

And this sort of this would be best commonly defined for the repo e.g. in Directory.build.props

```
...
  <PropertyGroup>
    <ProjectReunion_CsWinRT_Projections_RuntimeFrameworkVersion>10.0.18362.16</>
    <ProjectReunion_CsWinRT_Projections_TargetingPackVersion>10.0.18362.16</>
...
```

and use the common definitions across the repository

```
...
<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="$(ProjectReunion_CsWinRT_Projections_RuntimeFrameworkVersion)"/>
<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="ProjectReunion_CsWinRT_Projections_TargetingPackVersion" />
...
```

Even better: instead of defining versions in `Directory.Build.props`, create a new `Tools.props` in the root with the tool versions (similar to BuildInfo.json generated by the build pipeline) and `Directory.Build.props` merely includes it.

Reply:
those versions are wrong.
The SDK refs are probably unnecessary.

Reply:
@Scottj1s can we drop the `Microsoft.Windows.SDK.NET.Ref` snippets?

Reply:
remove the framework reference overrides in a component - those are needed by an app for a temporary situation to avoid mem leaks that were fixed in the .16 update. That will come to all apps by default in the next .NET5 SDK update